tests/e2e/config/config.yaml

```diff
@@ -6,13 +6,13 @@ operatorReplicas: 1
 noSetup: false
 artifactsDir: ../../_artifacts
 
-nginxVersion: controller-v1.9.1
+nginxVersion: controller-v1.9.6
 nginxURL: https://raw.githubusercontent.com/kubernetes/ingress-nginx/${NGINX_VERSION}/deploy/static/provider/kind/deploy.yaml
 
-certManagerVersion: v1.13.1
+certManagerVersion: v1.14.2
 certManagerChartURL: https://charts.jetstack.io/charts/cert-manager-${CERT_MANAGER_VERSION}.tgz
 
-rancherVersion: 2.7.8
+rancherVersion: 2.8.2
```

Comment:
Hehe, was trying to find 2.8.1, but I'm guessing it's a typo in the title?

Reply:
Yes, also confused myself here. :D